hack/istio-config-1.9.1.yaml

```diff
@@ -50,6 +50,6 @@ spec:
             value:
               name: ca-root-cert
               configMap:
-                name: istiod-tls
+                secretName: istiod-tls
```

Comment:
Thanks! I think this line needs to be removed completely?

Reply:
@JoshVanL I copied it from the old version: https://github.com/cert-manager/istio-csr/blob/dacb33e62cd33ff6f5787d371de35666d5d29511/hack/istio-config-1.8.2.yaml#L53

Reply:
Yep, I think it is also wrong there as well.

Reply:
So you want me to remove all three lines, or just the one for 1.9.1?